src/index.js

```diff
@@ -309,10 +307,10 @@ function graphqlHTTP(options: Options): Middleware {
           operationName, result
         });
         response.setHeader('Content-Type', 'text/html; charset=utf-8');
-        sendResponse(response, payload);
+        response.end(payload);
```

Comment:
I use `response.end` because is a method available in all the supported servers (restify, express, connect) and it's just enough to return a response which we know can only be a string, since this is in response of a GraphiQL request.

The main purpose of `sendResponse` is to deal with different types of data and address them properly with the best server method available to handle them.
Initially I just kept the use of `response.send` in this function, but I know see that it can be removed which would also mean that I can restore the use of `sendResponse` also for the GraphiQL response, will update the PR.